InvenTree/part/filters.py

```diff
@@ -71,6 +71,22 @@ def annotate_total_stock(reference: str = ''):
     )
 
 
+def annotate_last_updated(reference: str = ''):
```

Comment:
Question: What is `reference` used for?

Reply:
No idea.  `annotate_total_stock` had it, so I did the same.  As noted in the original post, I don't know if I need it because the code crashes but doesn't give me a stack trace.